jbehave-support-core/src/main/java/org/jbehavesupport/core/verification/VerificationSteps.java

```diff
@@ -39,6 +38,6 @@ private void verifyRow(Map<String, String> tableRow, SoftAssertions softly) {
     }
 
     private Verifier resolveVerifier(String verifierName) {
-        return (StringUtils.isNotEmpty(verifierName)) ? verifierResolver.getVerifierByName(verifierName) : equalsVerifier;
+        return verifierResolver.getVerifierByName(verifierName, VerifierNames.EQ);
```

Comment:
Another unnecessary call for resolving.

Reply:
We swapped the lines and the methods for greater readability